### ISSUE TITLE
ACC-253.1,ACC-270.1 (Level AA) _ Region _No label or tooltip provided and not reachable via keyboard

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3213]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	Region
Affected Check Point:	ACC-253.1, ACC-270.1
Short text:	ACC-253.1,ACC-270.1 (Level AA) _ Region _No label or tooltip provided and not reachable via keyboard

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	

3. Issue Type: Manual
Observed Behavior:
There is no visible label or tool tip or information provided for the ‘Preview’ region within the ‘YAML’ tab under the Step ‘Configuration’ in the dialog ‘Cluster Connect’.
And it is not reachable and accessible via keyboard.
Screenshot:
 
Screenshot 2:
 

Expected Behavior:
All UI elements should have visible and meaningful label provided and it should associate with the UI. Meaningful label supports the user in entering data correctly. 
[ACC-253.1,ACC-270.1 (Level AA) _ Region _No label or tooltip provided and not reachable via keyboard.docx](https://github.com/user-attachments/files/16686724/ACC-253.1.ACC-270.1.Level.AA._.Region._No.label.or.tooltip.provided.and.not.reachable.via.keyboard.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16686725/Issue.processing-09.2021.2.docx)
